snappy/install.go

```diff
@@ -46,10 +46,44 @@ const (
 	AllowOEM
 )
 
-// Update the installed snappy packages, it returns the updated Parts
+func doUpdate(part Part, flags InstallFlags, meter progress.Meter) error {
+	if _, err := part.Install(meter, flags); err == ErrSideLoaded {
+		logger.Noticef("Skipping sideloaded package: %s", part.Name())
+		return nil
+	} else if err != nil {
+		return err
+	}
+	if err := GarbageCollect(part.Name(), flags, meter); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Update updates the selected name
+func Update(name string, flags InstallFlags, meter progress.Meter) ([]Part, error) {
```

Comment:
Thanks! I did that now. It looks very similar. Which makes me assume you tell me that I should make api.go use the new snappy.Update() interface. I did that now. If that is not what you meant, please pardon me and let me know what you did mean :)